open-api-spec.yaml

```diff
@@ -691,7 +707,7 @@ components:
         incomingAmount:
           $ref: '#/components/schemas/amount'
         receivedAmount:
-          $ref: '#/components/schemas/amount'
+          $ref: '#/components/schemas/receivedAmount'
```

Comment:
Because the received amount can have a value of 0 whereas all other amounts should have a value greater than 0. If we don't want to enforce that on the spec level, I can change that such that only one amount type is necessary.

Reply:
I changed it to just 1 amount type with type string, format uint64.